### ISSUE TITLE
Installing Objection behind a corporate proxy on Linux

### ISSUE THREAD

**Issue by Simon-Davies** [sensepost/objection#71]
You can not install Objection behind a corporate proxy on Linux

It can be observed that a copy of certifi is downloaded to /tmp when running the command:

pip3 install objection --no-clean 

The problem with this is that the corporate proxy certificate is added to the cacert.pem file of the version of certifi already installed on the system but this isn't used and so an "SSL: CERTIFICATE_VERIFY_FAILED" Error is thrown by urllib. 

It would be good if there was an option to use the version of certifi already installed on the system.

Dirty Fix: Manually download: frida-10.7.7-py3.6-linux-x86_64.egg

There doesn't seem to be a download for Python 3.5 on the downloads page so I renamed frida-10.7.7-py3.6-linux-x86_64.egg to frida-10.7.7-py3.5-linux-x86_64.egg

**Comment by gerchicov-bp** [comment]
It seems I have the same issue with OSX:
```
error: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1056)>
    
    ----------------------------------------
Command "/Users/gerchicov/virtual-python3/bin/python3 -u -c "import setuptools, tokenize;__file__='/private/tmp/pip-install-nlu4mokq/frida/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /private/tmp/pip-record-yob586_3/install-record.txt --single-version-externally-managed --compile --install-headers /Users/gerchicov/virtual-python3/bin/../include/site/python3.7/frida" failed with error code 1 in /private/tmp/pip-install-nlu4mokq/frida/
```

**Comment by aph3rson** [comment]
@Simon-Davies @gerchicov-bp Are you able to install other pip packages, or is this just an issue with Objection? Looking at the setup.py that pip calls, there's no special logic there to do URL requests.

**Comment by leonjza** [comment]
There exists some extra download logic in the [frida-python](https://github.com/frida/frida-python/blob/f238498cdc6a605a2709f58098a054449e2ed8a1/setup.py) dependency.

**Comment by aph3rson** [comment]
This probably should be logged as an upstream issue in frida-python, then. Unfortunately, this is likely due to their usage of the requests library, which relies on the certifi library for certificate validation. There's some discussion on how pip fixed this issue in pypa/pip#2510.
